packages/honeycomb-opentelemetry-web/src/types.ts

```diff
@@ -88,6 +88,12 @@ export interface HoneycombOptions extends Partial<WebSDKConfiguration> {
    */
   spanProcessors?: SpanProcessor[];
 
+  /** Provide an array of exporters
+   * Use this to configure custom tracing services to export data in their own format
+   * E.g. You want to send data to another service.
+   */
+  traceExporters?: SpanExporter[];
+
```

Comment:
It might be good to call out in the comment that this is in _addition_ to the default exporter. One thing we talked about offline, was a config name that more clearly telegraphs the intent. Do folks have strong objections to `additionalTraceExporters`  to highlight that this is for providing more functionality?

Reply:
Oooo good catch.